numpy/random/_generator.pyx

```diff
@@ -3572,13 +3572,30 @@ cdef class Generator:
         >>> y.shape
         (3, 3, 2)
 
-        Here we generate a scatter plot of 800 samples from the bivariate
-        normal distribution with mean (0, 0) and covariance matrix
-        [[6, -3], [-3, 3.5]].
+        Here we generate 800 samples from the bivariate normal distribution
+        with mean (0, 0) and covariance matrix [[6, -3], [-3, 3.5]].  The
```

Comment:
should you use [ ] consitently, e.g. ,replace (0, 0) with [0, 0]

Reply:
Done.